src/godot_sqlite.cpp

```diff
@@ -298,7 +299,14 @@ void SQLite::_bind_methods() {
 			&SQLite::create_query);
 }
 
-bool SQLite::open(const String &path) {
+bool SQLite::open(const Ref<SQLiteDatabase> &database) {
```

Comment:
Not needed?

Reply:
var db: SQLiteDatabase = db.new()
var is_open_in_memory: bool = db.open_in_memory()
var query: SQLiteQuery = db.create_query("select 1;)